COMMPLAN.md

```diff
@@ -0,0 +1,239 @@
+#### What should we use it for?
+
+[The Xdev Twitter account](https://twitter.com/NCARXDev) should be for
+making *announcements* to the general public, such as making an
+announcement of a new blog post. In general, Twitter *could* be used
+for any announcement, but because of the limitations of the platform
+(i.e., limiting the length of the tweet) it is best to tie the tweet
+to *another* communication platform, such as a blog post.
+
+#### What should we *not* use it for?
+
+[The Xdev Twitter account](https://twitter.com/NCARXDev) should *not* be
+used for personal use.  Currently, only Kevin has the "keys" to the
+Twitter account.  If you want to make a broad announcement, contact Kevin
+via Zulip.
+
+### Email
+
+#### Why should we use it?
+
+The Xdev email address (`xdev@ucar.edu`) should be used only sparingly by
+any of us on the Xdev Team.  As a general rule, having the Xdev email address
+is useful for communication from *outside* of NCAR (or just *outside* of
+our Team) with the entire team.  It is also useful for creating Google
+Calendar meetings, as it allows the entire team to be invited to a meeting
+(and nobody is forgotten).
+
+#### What should we use it for?
+
+The Xdev email address should be used for sharing with outside entities or
+individuals as a means of reaching out to the entire team.  For example,
+we might want to share this email address with participants of our tutorials,
+or with users of our websites.  It should be used as the main point of
+contact with us for people who are *external to NCAR*.
+
+It is also incredibly handy to use the Xdev email address for inviting
+the entire team to Google Calendar events (see below), or for announcing
+Google Calendar event cancelations, etc.
+
+#### What should we *not* use it for?
+
+Avoid using the Xdev email address for internal communication.  Use Zulip
+or GitHub for collaboration within the team.
+
+### Google Calendar
+
+#### Why should we use it?
+
+Google Calendar is really the best way of coordinating times for meetings and
+any timed activities that we engage in as a team.  Together with the Xdev
+email address, it is the best way of reaching out to the entire team to see
+if they are available to meet at a specific time, on a specific day.
+
+#### What should we use it for?
+
+Use Google Calendar for scheduling all meetings.  Note that it can be difficult
+to find time for all people on the team to meet.  To accommodate this potential
+need, we have 2 standing "slots" on the UCAR Google Calendar called "Team Time".
+These slots are for *ad hoc* meetings with the entire time.  Having these
+regular meetings on the calendar, even if they are not used every week, lets
+everyone on the team know to try their best to keep those times free.  Then we
+have the best chance of everyone being available for any *ad hoc* meetings we
+might need to have.
+
+#### What should we *not* use it for?
+
+As usual, don't use the Google Calendar for person things.  Otherwise, use
```

Comment:
I think you want "personal things", not "person things." Also, this is the only place you refer to "the Google Calendar" rather than just "Google Calendar"